js/geo/coordinate.js

```diff
@@ -15,6 +15,7 @@ function Coordinate(column, row, zoom) {
     this.column = column;
     this.row = row;
     this.zoom = zoom;
+    this.posMatrix = null;
```

Comment:
The coords used in draw* are actually TileCoords `js/source/tile_coord.js`. I think this is meant to be part of TileCoord.

> It would make more sense to cache within the calculatePosMatrix method itself.

`calculatePosMatrix` should probably not be a `Painter` method. Where should it go?

The equivalent method in -native is `matrixFor` and it is part of `TransformState`. -native stores matrices on it's `Tile` object. In -js wrapped tiles have one shared `Tile` object. In -native there are two `Tile` objects with a shared `TileData`.